py4DSTEM/process/diffraction/crystal_phase.py

```diff
@@ -3,6 +3,7 @@
 from scipy.optimize import nnls
 import matplotlib as mpl
 import matplotlib.pyplot as plt
+from scipy.ndimage import gaussian_filter
 
 from emdfile import tqdmnd, PointListArray
 from py4DSTEM.visualize import show, show_image_grid
```

Comment:
Class names really shouldn't have underscores in them... I'm not sure why we didn't catch this earlier haha

Reply:
Good catch haha!